gov/politeia/proposals.go

```diff
@@ -279,21 +285,21 @@ func (db *ProposalDB) updateInProgressProposals() (int, error) {
 			continue
 		}
 
-		// 2. The new proposals status is NotAuthorized(has not changed).
-		// 3. The last update timestamp has not changed.
-		if proposal.Data.VoteStatus == statuses[0] || proposal.Data.Timestamp == val.Timestamp {
+		// Add the generated field.
+		proposal.Data.ID = val.ID
+
+		// 2. The new proposal data has not changed.
+		if reflect.DeepEqual(*proposal.Data, *val) {
```

Comment:
I'd like to keep `reflect.DeepEqual` in tests only.  Please make an `IsEqual` method instead.